Dockerfile

```diff
@@ -0,0 +1,13 @@
+# ---- Base images - order important ----
+FROM hmcts/ccd-definition-processor:latest as prcoessor
+FROM hmcts/ccd-definition-importer:latest as importer
```

Comment:
The above 2 base images contain useful scripts (which we're not using atm but will be) for automating config upload to PR etc